README.md

```diff
@@ -82,4 +81,4 @@ If a stream is being actively written to, the reader must take care to keep up w
 1. Blocking the entire connection until the offending stream is read.
 2. Resetting the offending stream.
 
-For example, the go-multiplex implementation blocks for a short period of time and then resets the stream if necessary.
+For example, the go-mplex implementation blocks for a short period of time and then resets the stream if necessary.
```

Comment:
@jacobheun I don't think that the js handles this in any way, and we potentially run into situations where one slow/staled reader/writer blocks the whole multiplexer. Something to look into and fix perhaps.

Reply:
Ticket added on the js side to keep track of this, https://github.com/libp2p/js-libp2p-mplex/issues/85